### PR TITLE
Add test to check name uniqueness

### PR DESCRIPTION
**Pull request by maximevaillancourt**
This PR adds a test that verifies that every record in the `.yml` file has a unique name.

### CODE REVIEW

**Comment by maximevaillancourt** [comment]
cc/ @joshnuss

File: test/unit/payment_icon_test.rb [hunk 1]
```diff
@@ -23,6 +23,12 @@ class PaymentIconTest < ActiveSupport::TestCase
     end
   end
 
+  test 'Every payment icon record has a unique "name"' do
+    names = PaymentIcon.all.map(&:name)
+    duplicates = names.detect {|e| names.rindex(e) != names.index(e) }
```
Comment (joshnuss) [inline test/unit/payment_icon_test.rb]: Good idea to add this!
Possible alternative assertion: `assert names.uniq.length < names.length`

Comment (joshnuss) [reply test/unit/payment_icon_test.rb]: Though, maybe then you wouldn't know the duplicates..

Comment (maximevaillancourt) [reply test/unit/payment_icon_test.rb]: We wouldn't be able to identify the duplicates though (we'd only _know_ there are duplicate entries). 🤔

**Pull request merged**